Update Design Standards

okkays opened the pull request:
Implements the redesign of the [design standards page](https://www.ada.gov/law-and-regs/design-standards/).

- 🌎 The design standards don't currently include images
- ⛔ This makes it hard to see at-a-glance what document you're
  downloading or page you're going to.
- ✅ This PR adds images and groups standards with guidance to make documents distinguish to find at-a-glance.

Before:

<img width="1081" alt="Previous design standards (gif demo - see federalist preview link)" src="https://user-images.githubusercontent.com/15126660/215852963-6dbc8565-0f15-4c78-8f39-30fc14f8d9d5.png">

After:

![New design standards (gif demo - see federalist preview link)](https://user-images.githubusercontent.com/15126660/215875800-e5b19ae9-5ffc-4702-b2f9-add01b0725be.gif)

nguyen-tina reviewed (commented):
This is mostly looking good – one nit: Is it possible to make the icon + download text flush to the line above? There's currently 16px margin that's not needed.

nguyen-tina commented:
Also, strangely, the icon for the Guidance on the 2010 ADA Standards for Accessible Design is smaller than the other two icons...

c2nelson commented:
I think this looks great.  I agree with @nguyen-tina re tweaking the margin.

okkays commented:
> This is mostly looking good – one nit: Is it possible to make the icon + download text flush to the line above? There's currently 16px margin that's not needed.

Margin removed, replaced with an 8px gap that should only appear between the icon and text (and anything else we'd ever decide to add to the link)

> Also, strangely, the icon for the Guidance on the 2010 ADA Standards for Accessible Design is smaller than the other two icons...

This was a consequence of flexbox thinking that the text was more important than the icon - nice catch, fixed!

kewitham commented on _assets/sass/custom/_buttons.scss:
```diff
@@ -35,10 +35,11 @@
   align-items: baseline;
   display: inline-flex;
-}
+  gap: 8px;
```
I didn't know this was a thing! Super useful!

okkays replied:
😄 [I definitely recommend this guide](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) - it's definitely worth skimming the pictures, I go back to it pretty much every time I use flex.

nguyen-tina replied:
these pics are so useful

nguyen-tina commented on _assets/sass/custom/_page.scss:
```diff
@@ -66,3 +66,31 @@
+      margin-bottom: 1em;
+      max-width: 50%;
+      @include at-media-max(mobile-lg) {
+        max-width: 100%;
```
Agreed that keeping them at 50% is good for the user!

okkays replied:
Done!

Had a quick thought about how it'd look center-aligned, but left still looks better IMO

current:

<img width="200" alt="screenshot with left-aligned cover photo" src="https://user-images.githubusercontent.com/15126660/216116834-93434cf5-e606-49ad-9e9b-bd1d80ec1d02.png">

centered:

<img width="200" alt="screenshot with center-aligned cover photo" src="https://user-images.githubusercontent.com/15126660/216116782-84c42e5f-6ce3-4f8f-b270-71cd95a27d68.png">

nguyen-tina replied:
great job on moving quickly Kevin! 

they both look good, and at this point it's preference – I think we should keep it consistent with what it is on desktop, even though the center-alignment is so tempting!

okkays commented:
@vwilliamsn Related content removed!
